apps/explorer/src/ui/SplitPanes.tsx

```diff
@@ -20,13 +20,13 @@ function ResizeHandle({ isHorizontal }: { isHorizontal: boolean }) {
 
     return (
         <PanelResizeHandle
-            className={clsx('group', isHorizontal ? 'px-2' : 'py-2')}
+            className={clsx(isHorizontal ? 'px-2' : 'py-2')}
```

Comment:
This is an intentional removal of the group stuff?

Reply:
removed, unintended change